protected/humhub/modules/content/widgets/stream/WallStreamEntryWidget.php

```diff
@@ -139,6 +146,21 @@ abstract class WallStreamEntryWidget extends StreamEntryWidget
      */
     protected $renderOptionClass = WallStreamEntryOptions::class;
 
+    /**
+     * @var string Action for menu of create form on wall stream
+     */
+    public $createFormMenuAction;
+
+    /**
+     * @var int Sort order of create form and tab menu on wall stream
+     */
+    public $createFormSortOrder;
+
+    /**
+     * @var string Class name of the Form to create a Content from wall stream
+     */
+    public $createFormClass;
```

Comment:
Must/Should this class based on `WallCreateContentForm`? If yes we should add this to the comment.

Reply:
Yes, it must, I added this in commit https://github.com/humhub/humhub/pull/5827/commits/8638d49b492a58fb7135705b49501166c094c8ca.